catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/web/metrics/IcebergMetricsFormatter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2023 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics;
+
+import com.datastrato.gravitino.catalog.lakehouse.iceberg.web.IcebergObjectMapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.iceberg.metrics.MetricsReport;
+
+public class IcebergMetricsFormatter {
+
+  private ObjectMapper icebergObjectMapper;
+
+  IcebergMetricsFormatter() {
+    this.icebergObjectMapper = IcebergObjectMapper.getInstance();
+  }
+
+  public String toPrintableString(MetricsReport metricsReport) {
+    try {
+      return toJson(metricsReport);
+    } catch (JsonProcessingException e) {
+      return metricsReport.toString();
+    }
+  }
+
+  public String toJson(MetricsReport metricsReport) throws JsonProcessingException {
+    return icebergObjectMapper.writeValueAsString(metricsReport);
+  }
+}
```

Comment:
If this class is only used for print logs, I don't think it is necessary to have a class, some helper methods should be enough.

Reply:
I will use this class to serialize metrics report to json string in KVMetricsStorage in next pr. should I keep it as a class?